src/devices/Bmx280/BmxBase.cs

```diff
@@ -13,13 +13,14 @@
 
 namespace Iot.Device.Bmx280
 {
-    public class BmxBase : IDisposable
+    public abstract class BmxBase : IDisposable
     {
         internal I2cDevice _i2cDevice;
         internal byte _deviceId;
         internal bool _initialized = false;
         internal CommunicationProtocol _communicationProtocol;
-        internal CalibrationData _calibrationData;
+
+        internal CalibrationData CalibrationData { get; private set; }
```

Comment:
Why do we want to change it to a property instead of keeping it as a member of the class?

Reply:
I wanted `set` restrictions on this field (hence the property refactor). 

The base class owns `_calibrationData`, but implementations need read access too. Since the `Init` operation that creates  `_calibrationData` is both owned by `BmxBase` and is lazily initialized by it, it made sense to me to refactor `_calibrationData` to be a property on `BmxBase`.

Reply:
oh ok, that makes sense.